integr8ly/integreatly-operator
# feat: adds mdc install

This PR adds the installation of the mobile developer console as a product of the integreatly operator.

https://issues.jboss.org/browse/INTLY-3290

Verification steps:
1) Make sure the MDC operator manifest file has been pushed to the integreatly registry in [quay.io](https://quay.io/application/) 
2) Check if the MDC operator is available on the operator hub under `Catalog >> Operator Hub`
3) Run the integreatly-operator locally installing only the MDC operator: 
`operator-sdk up local --namespace=<integreatly_operator_namespace> --operator-flags=--products=mdc`
4) Verify if the `integreatly-mdc` namespace is created
5) Check if there is a subscription of the MDC operator in the integreatly-mdc namespace under `Catalog >> Operator Management` and if the status is Up to date
6) In the `integreatly-mdc` namespace check if there are two pods created, one for the MDC operator and another for the MDC service
